### ISSUE TITLE
Could not find com.facebook.flipper:flipper:0.91.2.

### ISSUE THREAD

**Issue by a-eid** [facebook/flipper#2367]
## 🐛 Bug Report
Could not find com.facebook.flipper: flipper:0.91.2.

## To Reproduce
update to 0.91.2 

## Environment
react-native: 64.1

```
* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.facebook.flipper:flipper:0.91.2.
     Searched in the following locations:
       - file:/Users/aeid/.m2/repository/com/facebook/flipper/flipper/0.91.2/flipper-0.91.2.pom
       - file:/Users/aeid/code/zeal-app-reactnative/node_modules/react-native/android/com/facebook/flipper/flipper/0.91.2/flipper-0.91.2.pom
       - file:/Users/aeid/code/zeal-app-reactnative/node_modules/jsc-android/dist/com/facebook/flipper/flipper/0.91.2/flipper-0.91.2.pom
       - https://dl.google.com/dl/android/maven2/com/facebook/flipper/flipper/0.91.2/flipper-0.91.2.pom
       - https://jcenter.bintray.com/com/facebook/flipper/flipper/0.91.2/flipper-0.91.2.pom
       - https://www.jitpack.io/com/facebook/flipper/flipper/0.91.2/flipper-0.91.2.pom
     Required by:
         project :app
         project :app > project :react-native-flipper
   > Could not find com.facebook.flipper:flipper-network-plugin:0.91.2.
     Searched in the following locations:
       - file:/Users/aeid/.m2/repository/com/facebook/flipper/flipper-network-plugin/0.91.2/flipper-network-plugin-0.91.2.pom
       - file:/Users/aeid/code/zeal-app-reactnative/node_modules/react-native/android/com/facebook/flipper/flipper-network-plugin/0.91.2/flipper-network-plugin-0.91.2.pom
       - file:/Users/aeid/code/zeal-app-reactnative/node_modules/jsc-android/dist/com/facebook/flipper/flipper-network-plugin/0.91.2/flipper-network-plugin-0.91.2.pom
       - https://dl.google.com/dl/android/maven2/com/facebook/flipper/flipper-network-plugin/0.91.2/flipper-network-plugin-0.91.2.pom
       - https://jcenter.bintray.com/com/facebook/flipper/flipper-network-plugin/0.91.2/flipper-network-plugin-0.91.2.pom
       - https://www.jitpack.io/com/facebook/flipper/flipper-network-plugin/0.91.2/flipper-network-plugin-0.91.2.pom
     Required by:
         project :app
   > Could not find com.facebook.flipper:flipper-fresco-plugin:0.91.2.
     Searched in the following locations:
       - file:/Users/aeid/.m2/repository/com/facebook/flipper/flipper-fresco-plugin/0.91.2/flipper-fresco-plugin-0.91.2.pom
       - file:/Users/aeid/code/zeal-app-reactnative/node_modules/react-native/android/com/facebook/flipper/flipper-fresco-plugin/0.91.2/flipper-fresco-plugin-0.91.2.pom
       - file:/Users/aeid/code/zeal-app-reactnative/node_modules/jsc-android/dist/com/facebook/flipper/flipper-fresco-plugin/0.91.2/flipper-fresco-plugin-0.91.2.pom
       - https://dl.google.com/dl/android/maven2/com/facebook/flipper/flipper-fresco-plugin/0.91.2/flipper-fresco-plugin-0.91.2.pom
       - https://jcenter.bintray.com/com/facebook/flipper/flipper-fresco-plugin/0.91.2/flipper-fresco-plugin-0.91.2.pom
       - https://www.jitpack.io/com/facebook/flipper/flipper-fresco-plugin/0.91.2/flipper-fresco-plugin-0.91.2.pom
     Required by:
         project :app
         ```

**Comment by radko93** [comment]
I'm also having this issue. I've tried 0.90.0 and it's the same.

**Comment by sharplet** [comment]
Looks like the latest version on Maven central is 0.91.1: https://mvnrepository.com/artifact/com.facebook.flipper/flipper.

**Comment by ankur-malviya** [comment]
How was it fixed ? I am having same issue

**Comment by mweststrate** [comment]
Maven pipeline was unblocked and artificats are available now: https://mvnrepository.com/artifact/com.facebook.flipper/flipper. For any new issues please open a new issue :)

**Comment by joeljerushan** [comment]
Can anyone tell how to fix this issue or downgrade flipper to compatible version ?

**Comment by mweststrate** [comment]
https://fbflipper.com/docs/getting-started/react-native#using-the-latest-flipper-sdk

**Comment by ankur-malviya** [comment]
> Can anyone tell how to fix this issue or downgrade flipper to compatible version ?

Earlier `FLIPPER_VERSION` was `0.37.0 `but it couldn't be found on this repo - 
https://repo.maven.apache.org/maven2/com/facebook/flipper/flipper/

So I changed my `FLIPPER_VERSION` to `0.93.0` and its working now.
`FLIPPER_VERSION` can be changed from `android/gradle.properties`

**Comment by Parth-9711** [comment]
Can anyone update ?

**Comment by daryazata** [comment]
Im also facing this issue with 0.99.0 version!

**Comment by trickeyd** [comment]
I'm also having this with 99 having just upgraded from react native 61.5 to 66.2

**Comment by trickeyd** [comment]
Tried 118.1, 110, 98, 100, 101 as well a 93 mentioned above as working. 🤷 

What was it Einstein didn't say again? 
"The definition of insanity is doing the same thing over and over again and expecting a different result"

Guess I'll look for some other solutions. If anyone above has an idea, please do let me know.

**Comment by aaronfg** [comment]
Having the same issue after upgrading to RN 66.3. All flipper versions listed above aren't working.

**Comment by mweststrate** [comment]
Per https://mvnrepository.com/artifact/com.facebook.flipper/flipper 0.119 is available just fine. Please check mvnrepository first before reporting. Locking this issue as this one was resolved already, so please open new issues for new issues. Same symptoms != same issue.
